app/assets/javascripts/host_checkbox.js

```diff
@@ -63,7 +63,7 @@ $(function() {
 });
 
 function removeForemanHostsCookie() {
-  $.cookie($.cookieName, null);
+  $.removeCookie($.cookieName, { path: '/' });
```

Comment:
Yes, I see that. I was curious about what path it'd have when Foreman isn't at the root, since the removeCookie documentation states:

<pre>
Note: when deleting a cookie, you must pass the exact same path, domain and secure options that were used to set the cookie, unless you're relying on the default options that is.
</pre>

And as far as I can see, we don't specify a path during creation.  (Which I suppose means it'd be the "basename" of the page URL, or the page URL itself?)